test/passing/verbose2.ml.ref

```diff
@@ -1,6 +1,6 @@
-project-root={CWD}
-margin=77 ({CWD}/.ocamlformat:1)
-wrap-comments=true ({CWD}/.ocamlformat:2)
+project-root=../../ocamlformat
```

Comment:
The previous value was wrong (the test is executed from the `test` directory, not its parent). I think the excessive `../ocamlformat` is due to `relativize` and using `normalize` before or after does not fix this.